src/components/RequestModal/AdvancedRequester/index.tsx

```diff
@@ -365,24 +365,27 @@ const AdvancedRequester = ({
                   )}
                   {!isValidating &&
                     serverData &&
-                    serverData.profiles.map((profile) => (
-                      <option
-                        key={`profile-list${profile.id}`}
-                        value={profile.id}
-                      >
-                        {isAnime &&
-                        serverData.server.activeAnimeProfileId === profile.id
-                          ? intl.formatMessage(messages.default, {
-                              name: profile.name,
-                            })
-                          : !isAnime &&
-                            serverData.server.activeProfileId === profile.id
-                          ? intl.formatMessage(messages.default, {
-                              name: profile.name,
-                            })
-                          : profile.name}
-                      </option>
-                    ))}
+                    serverData.profiles
+                      .slice()
+                      .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
You could use `toSorted` instead of having to create a copy, then call sort

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted